src/main/java/pl/edu/agh/xp/Main.java

```diff
@@ -1,9 +1,9 @@
 package pl.edu.agh.xp;
 
-
 public class Main {
 
-    public static void main(String[] args) {
+    public static void main(String[] args)
+    {
```

Comment:
According to coding rule no. 3 we should use default IntelliJ coding style so no enter after method name